src/exodus/rpctx.cpp

```diff
@@ -1570,6 +1575,18 @@ UniValue exodus_sendmint(const UniValue& params, bool fHelp)
 
         denoms.insert(denoms.end(),
             static_cast<size_t>(amount), static_cast<uint8_t>(denomId));
+
+        int confirmations;
+        try {
+            confirmations = _my_sps->getDenominationConfirmation(propertyId, denomId, minConfirms);
+        } catch (std::invalid_argument const &e) {
+            throw JSONRPCError(RPC_INVALID_PARAMETER, e.what());
+        }
+
+        if (confirmations != -1 && confirmations < minConfirms) {
```

Comment:
We need to revise this condition if we change the behavior of `getDenominationConfirmation`.

Reply:
have changed, just throw if remaining block is not zero.